client/inbox-panel/style.scss

```diff
@@ -176,6 +176,10 @@
 	}
 }
 
+.woocommerce-layout__activity-panel-content {
+	padding: $gap-large 0;
```

Comment:
I think this rule adds extra padding to the bottom of the container. Could `padding-top: $gap-large` be used instead? 

![Screen Shot 2020-11-27 at 12 41 05 pm](https://user-images.githubusercontent.com/9312929/100411962-34033400-30ae-11eb-9f6e-b26b21776742.png)

Reply:
@adrianduffell Thank you Adrian 👍 Nice catch! Change is made in https://github.com/woocommerce/woocommerce-admin/commit/954d8b47f995e14cdc5b1605997090eb46b6552d

Reply:
@adrianduffell It looks like I need your approval as well to merge this PR. Could you approve it, please 😃 ?